src/backend/distributed/planner/multi_explain.c

```diff
@@ -1482,11 +1495,18 @@ WrapQueryForExplainAnalyze(const char *queryString, TupleDesc tupleDesc)
 	}
 
 	StringInfo explainOptions = makeStringInfo();
-	appendStringInfo(explainOptions, "{\"verbose\": %s, \"costs\": %s, \"buffers\": %s, "
-									 "\"timing\": %s, \"summary\": %s, \"format\": \"%s\"}",
+	appendStringInfo(explainOptions,
+					 "{\"verbose\": %s, \"costs\": %s, \"buffers\": %s, "
+#if PG_VERSION_NUM >= PG_VERSION_13
+					 "\"wal\": %s, "
+#endif
+					 "\"timing\": %s, \"summary\": %s, \"format\": \"%s\"}",
```

Comment:
Maybe we could use `quote_qualified_identifier` instead of quoting the strings manually, you might have a look at the function's usage in codebase